002_simulation_structure/002_simulation_structure.tex

```diff
@@ -1,23 +1,109 @@
+      \item  Food Taking – people are allowed to take as much food from the platform as they are able to eat. They cannot preserve food to eat on separate days. Additionally, the utility of the food taken decreases exponentially (each unit of food has a decreasing improvement to HP) – representing the law of diminishing returns and acting as a deterrent to basic strategies.
+      \item A sufficiently honest environment – people are generally truthful, direct deceit does not exist, only the ability to ignore. There are very specific conditions in which deceit is permitted, these will be discussed alongside the introduction of the relevant features, but is related to breaking agreements in times of extreme need.
+
+\end{enumerate}
+
+\subsection{Observations \& Understanding}
+
+An initial observation of the context immediately presents the concept of a World, the tower, and Agents, the people. These agents interact with one another and the tower, within constraints, and must be able to react and evolve their behaviour to maximise their utility and survival.
+ 
+The tower itself has several laws that define the simulation overall, for example the uniform movement of the platform and the food level control. The implementation of these are relatively straightforward and there is not notable room for interpretation.
+ 
+Within the tower there exists a food allocation problem, which is directly related to the survival of agents and forms the core incentive for self-organisation. With a lack of organisation, the tower society would fall victim to the tragedy of the commons – the common pool resource would deplete, and the overall well-being of society would suffer at the expense of a few individuals’ gain. Hence, the crux of this project is a collective action resource management problem, and agents must be equipped with sufficient capabilities and information.
+ 
+The aforementioned capabilities can be split into two sections; those related to the tower and those related to the agent interactions.
+ 
+Within the tower, agent abilities are listed below:
+
+\begin{enumerate}
+    \item View Food on Platform: The agent cannot view the food reserves of the platform unless it is at the agent’s floor or the floor below the agent.
+    \item Food Consumption: Agent’s cannot preserve food but the option to ‘destroy’ food (eat food without receiving notable utility) is valid. Agents can only eat food from the platform when it is at the agent’s floor.
+    \item Time Details: Agents do not have explicit access to timed parameters (reshuffle period, day length, time of platform on each floor etc.), however, can use memory to calculate these.
+    \item Food Details: Agents do not know the food on the platform at the beginning of each day.
+
+    \item Health Details: Agents do have a sense of their current health level and utility they are receiving from food, as well as how long they can go without food before dying.
+    
+\end{enumerate}
```

Comment:
maybe standardize the use of - and : for your enumerate blocks? previous section uses -

Reply:
Done X